test/evaluation/subjectIsIssuerEvaluationHandler.spec.ts

```diff
@@ -14,112 +14,83 @@ describe('evaluate', () => {
 
   it('should return ok if subject_is_issuer is verified', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-subject-is-issuer.json').presentation_definition;
-    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-subject-is-issuer.json');
     const evaluationClient: EvaluationClient = new EvaluationClient();
     const subjectIsIssuerEvaluationHandler: SubjectIsIssuerEvaluationHandler = new SubjectIsIssuerEvaluationHandler(evaluationClient);
-    const presentationSubmission: PresentationSubmission = {
+    subjectIsIssuerEvaluationHandler.presentationSubmission = {
       'id': '3cpLWMyiAT1qQXTaJNWOG',
       'definition_id': '31e2f0f1-6b70-411d-b239-56aed5321884',
       'descriptor_map': [
         {
           'id': '867bfe7a-5b91-46b2-9ba4-70028b8d9cc8',
           'format': 'ldp_vc',
-          'path': '$.verifiableCredential[0]'
+          'path': '$[0]'
         }
       ]
     };
     const verifiableCredential: Array<VerifiableCredential> = [{
       'credentialSubject': {
-        'id': 'did:example:123'
+        'id': 'did:example:123',
+        'age': 19
       },
       'id': '2dc74354-e965-4883-be5e-bfec48bf60c7',
       'type': ['VerifiableCredential'],
-      '@context': [],
-      'issuer': '',
-      'issuanceDate': '',
+      '@context': ['https://www.w3.org/2018/credentials/v1'],
+      'issuer': 'did:example:123',
+      'credentialSchema': [{
+        'id': 'https://www.w3.org/TR/vc-data-model/#types'
+      }],
+      'issuanceDate': '02/11/2021',
```

Comment:
take a look at https://datatracker.ietf.org/doc/html/rfc3339#section-5.8

Reply:
The requested action is not very clear to me, either the response or the request it self could be more clear.

For example currently I want to mark it resolved but I do not have enough information on this conversation to proceed.

Reply:
The requested action is not very clear to me, either the response or the request it self could be more clear.

For example currently I want to mark it resolved but I do not have enough information on this conversation to proceed.

Reply:
@hrehman-sphereon There are many date formats that we will need to support in the future for v2 that are on that document. This is not even in the scope of my ticket although I updated two test cases to use one of the formats. No need to do anything about it at the moment.